src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/ManifestReader.cs

```diff
@@ -86,7 +86,9 @@ private static void ReadMetadataValue(ManifestMetadata manifestMetadata, XElemen
                         manifestMetadata.Id = value;
                         break;
                     case "version":
-                        manifestMetadata.Version = NuGetVersion.Parse(value);
+                        NuGetVersion version;
+                        NuGetVersion.TryParse(value, out version);
```

Comment:
It doesn't matter in this case, because our TryParse always resets version to `null`, but it bothers me when return values are ignored 🤷‍♂️

Reply:
I'm not sure if I understand your concern, `Parse` will return an error if it's unable to create the NuGetVersion, `TryParse` returns null. Are saying that `version` is ignored?

Reply:
> `TryParse` returns null. 

TryParse returns a non-nullable boolean. It's supposed to be used as an indication whether the parse was successful or not.

I believe this is the right reference, but I can't find any TryParse methods in our repo with a nullable return type.
https://github.com/NuGet/NuGet.Client/blob/8a5348f76ab0d2338dad0555f4abd7b59598c793/src/NuGet.Clients/NuGet.VisualStudio/LegacyTypes/SemanticVersion.cs#L156

Reply:
I understand now, I updated the code, so I use the return value